app/models/events/pull_request.rb

```diff
@@ -43,5 +45,15 @@ class PullRequest < ApplicationRecord
               :locked,
               inclusion: { in: [true, false] }
     validates :github_id, uniqueness: true
+
+    ##
+    # Returns the number of reviews this pull_request has
+    delegate :count, to: :reviews, prefix: true
```

Comment:
To get the number of `reviews` of a `pull_request` we need an instance of a `pull_request` and then `instance_of_pr.reviews.count`, in this case we allow doing something like this `Events::PullRequest.count_reviews` which will give us the amount of reviews that exists in the database, that will be the same as `Events::Reviews.count`.